api/base/utils.py

```diff
@@ -26,8 +27,8 @@
 # These values are copied from rest_framework.fields.BooleanField
 # BooleanField cannot be imported here without raising an
 # ImproperlyConfigured error
-TRUTHY = set(('t', 'T', 'true', 'True', 'TRUE', '1', 1, True, 'on', 'ON', 'On', 'y', 'Y', 'YES', 'yes'))
```

Comment:
This should require some discussion but I thought this was better to just be accurate to the comment.